public/views/index.html

```diff
@@ -67,22 +67,10 @@ <h1 class="red-text"><i class="fa fa-warning"></i></h1>
               <p>This game contains offensive, profane, explicit content created by <a href="#" >Cards Against 
                 Humanity</a> and provided here, uncensored and complete, in digital form. Human Resources made us 
                 say that.
+                {{dinoMessage}}
```

Comment:
Please use a better variable name other than dinoMessage

Reply:
This probably came from a pull request I made on Sat. Note: I worked only on a dashboard UI, and a dashboard controller.

Reply:
This is in your PR so you need to update it.

Reply:
I removed that line as soon as I saw it

Reply:
If you remove it, why is it still here?